www/class/centreonAuth.SSO.class.php

```diff
@@ -527,24 +537,29 @@ public function refreshToken(
         string $clientId,
         string $clientSecret,
         string $refreshToken,
+        bool $clientBasicAuth,
```

Comment:
can you also modify function definition?

Reply:
Sorry, added the function documentation for this parameter